src/netex_idf/lines.rs

```diff
@@ -116,9 +116,9 @@ fn load_netex_lines(
                     .map(Element::text)
                     .ok();
                 let private_code = line.only_child("PrivateCode").map(Element::text);
-                let network_id: String = line
-                    .try_only_child("RepresentedByGroupRef")?
-                    .try_attribute("ref")?;
+                let network_id: String = skip_fail!(line
```

Comment:
How about adding a test for that one?

Reply:
I thought about doing it yesterday. I just wrote that before leaving.